sources/lib/util/steps.ml

```diff
@@ -0,0 +1,73 @@
+open Options
+open Format
+
+type incr_kind = Matching | Omega | Fourier | Uf | Builtin | Ac | Naive of int
```

Comment:
Some internal explanations of what each step kind refers to would be useful, for instance what does `Naive 3` means ?